lib/archethic_web/domain.ex

```diff
@@ -80,8 +80,8 @@ defmodule ArchethicWeb.Domain do
           |> Application.get_env(ArchethicWeb.Endpoint)
           |> Keyword.fetch!(:https)
 
-        keyfile = Keyword.fetch!(https_conf, :keyfile)
-        certfile = Keyword.fetch!(https_conf, :certfile)
+        keyfile = Application.app_dir(:archethic, Keyword.fetch!(https_conf, :keyfile))
```

Comment:
I don't think you should do this change 
If someone wants to use a true certificate, for example cerbot/let's encrypt manages it in /etc folder. Hence we should let the file location as mentioned in the conf.

If you need to define the app_dir for self signed, we could do it in the conf itself.

Reply:
Yes right, the problem is that we cannot use `Application.app_dir(:archethic)` in config file as the application archethic do not exists yet.
And we need to use it because when we do a release, the application change it's folder depending of version number.
I don't find a way to create the complete path of the file during application compilation so I just added a case to read the file and if it's not found, add the app_dir and try again